cs/src/Management/TunnelManagementClient.cs

```diff
@@ -728,6 +729,13 @@ private string UserLimitsPath
                             ex.SetAuthenticationSchemes(authHeaderValues);
                         }
 
+                        // Propagate failed policy requirement names.
+                        if (response.Headers.TryGetValues(
+                            EnterprisePolicyFailureHeaderName, out var policyFaiulreValues))

```

Comment:
Super nit but failure is spelled wrong in the `policyFaiulreValues` variable name